### PR TITLE
Exporting default middlewares to a new folder and adding json support…

### PR DESCRIPTION
**Pull request by JoaoBGusmao**
… on 'where'

### CODE REVIEW

File: middlewares/sequelize-options.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    if (req.query.populate) {
+        options.include = req.query.populate.split(',');
+        delete options.where.populate;
+    }
+
+    if (req.query.fields) {
+        options.attributes = req.query.fields.split(',');
+        delete options.where.fields;
+    }
+
```
Comment (luizguilhermesj) [inline middlewares/sequelize-options.js]: this two `if` statments could be isolated methods, and the `delete` operation is now useful when there is a `req.query.where`

File: middlewares/sequelize-options.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module.exports = function (req, res, next) {
+    let where = req.query.where ? JSON.parse(req.query.where) : req.query
+
+    let options = {
+        where
+    }
```
Comment (luizguilhermesj) [inline middlewares/sequelize-options.js]: could it be a method that returns the options abstracting this logic?

**Review by luizguilhermesj — CHANGES_REQUESTED** [review]
Missing `express-routes.js` changes, that will be needed for this to work.

Also need to update the `readme` explaining how to use the new functionality and to update the package json with the new version as a major release (3.1.0)

**Comment by JoaoBGusmao** [comment]
changes done @luizguilhermesj

**Review by luizguilhermesj — APPROVED** [review]

**Pull request merged**